0_Fib/fib_test.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
```

Comment:
Add another function `TestFibonacci` that works with slices (_unit_ tests).
Include corner cases: `n := 1 | 0  | -1 | overflow-big`

Reply:
done